src/core_atmosphere/CMakeLists.txt

```diff
@@ -76,24 +89,55 @@ set(ATMOSPHERE_CORE_PHYSICS_WRF_SOURCES
     module_sf_noahlsm.F
     module_sf_noahlsm_glacial_only.F
     module_sf_oml.F
+    module_sf_ruclsm.F
     module_sf_sfcdiags.F
+    module_sf_sfcdiags_ruclsm.F
     module_sf_sfclay.F
     module_sf_sfclayrev.F
     module_sf_urban.F
+    module_sf_mynn.F
+    module_sf_myjsfc.F
     bl_mynn_post.F
     bl_mynn_pre.F
-    module_bl_mynn.F
-    module_cu_kfeta.F
-    module_cu_ntiedtke.F
-    module_cu_tiedtke.F
-    module_ra_cam.F
-    module_sf_mynn.F
```

Comment:
@guoqing-noaa we are trying as much as possible to not change the ncar code, even though it may make sense to organize differently; this will help us in future merges, from what I see in your changes there should only be one removal from the ncar code (since the GF module was renamed), so these six lines should be reverted

Reply:
@barlage Great point! I have not think about this. Thanks for the suggestion!